challenges/objects-arrays.js

```diff
@@ -89,37 +131,60 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+zooAnimals.forEach(element => {
+  animalNames.push(
+    `Name: ${element.animal_name}, Scientific: ${element.scientific_name}`
+  );
+});
 console.log(animalNames);
 
+
 /* Request 2: .map()    
 
+MDN
+const map1 = array1.map(x => x * 2);
+
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map((zooAnimal, index, zooAnimals) => {
+  return zooAnimal.animal_name.toLowerCase();
+});
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
+MDN
+const result = words.filter(word => word.length > 6);
+
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter(zooAnimal => {
+  if (zooAnimal.population <= 5) {
+    return true
+  };
```

Comment:
.filter() returns an element based on a true/false statement. 
```
const largerPopulation = zooAnimals.filter(zooAnimal => 
    zooAnimal.population <= 5
);
```
or without the `implicit return`
```
const largerPopulation = zooAnimals.filter(zooAnimal => {
    return zooAnimal.population <= 5;
});
```